### DISCUSSION TITLE
使用中文的数据进行重新训练，模型还能具有处理英文的能力吗?处理英文的效果如何呢？

### DISCUSSION THREAD

**Discussion by Nanamumuhan** [ymcui/Chinese-LLaMA-Alpaca#860] [General]
使用中文的数据进行重新训练，模型还能具有处理英文的能力吗?处理英文的效果如何呢？

**Comment by ymcui** [comment]
不清楚你说的“重新训练”是什么意思，随机初始化？在原版llama上继续训练？
只要不是随机初始化，英文能力是会有保留的，但也得看模型对你的中文数据的拟合情况。

**Reply by Nanamumuhan** [reply]
就是在现有的英语预训练模型上fine tuning一个新的中文的模型，模型参数在训练过程中会发生改变。

**Reply by Nanamumuhan** [reply]
后者是使用英文预训练模型初始化中文的预训练模型后在训练中文的模型，这样的花中文的预训练模型还能保留原有的英文的能力吗？

**Reply by ymcui** [reply]
你只训中文那么英文能力肯定是要丢失一些的，如果不想出现灾难性遗忘问题可以尝试同时使用中英文数据。

**Reply by Nanamumuhan** [reply]
那是不是得使用中英文对齐数据
